meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2022.10.inc

```diff
@@ -13,20 +13,20 @@ DESCRIPTION = "U-Boot"
 SECTION = "bootloaders"
 
 LICENSE = "GPL-2.0"
-LIC_FILES_CHKSUM = "file://Licenses/README;md5=5a7450c57ffe5ae63fd732446b988025"
+LIC_FILES_CHKSUM = "file://Licenses/README;md5=2ca5f2c35c8cc335f0a19756634782f1"
 PE = "1"
 
 SRC_URI = "git://github.com/u-boot/u-boot.git;branch=master;protocol=https"
-		
-SRCREV = "e4b6ebd3de982ae7185dbf689a030e73fd06e0d2"
+
+SRCREV = "4debc57a3da6c3f4d3f89a637e99206f4cea0a96"
```

Comment:
So, for every recipe the Q is the same, can we use named tags instead of putting SHAs?

Reply:
Yes, we can. Yocto manual says it is slight slower due to first needing to retrieve the SHA. What I would actually prefer is to put version in the recipe name (which translates into variable PV) and make future updating easier. As is, this is how it was structured in initial image recipe. How about we postpone this to Kirkstone?